src/types/v5-shared.ts

```diff
@@ -151,6 +153,16 @@ export type StopOrderTypeV5 =
  */
 export type PositionIdx = 0 | 1 | 2;
 
+/**
+ * Position status.
+ * 
+ * - 'Normal'
+ * - 'Liq' in the liquidation progress
+ * - 'Adl' in the auto-deleverage progress
+ */
+export type PositionStatusV5 = 'Normal' | 'Liq' | 'Adl';
+export type PositionSideV5 = 'Buy' | 'Sell';
```

Comment:
For position side v5 it should also include "none". Seems empty string is also an option, so perhaps `'Buy' | 'Sell' | 'None' | ''`:
<img width="1014" alt="Screenshot 2024-04-26 at 10 37 43" src="https://github.com/tiagosiebler/bybit-api/assets/14090872/6f65484a-3f7c-4335-b832-b1a1e4f16b6a">

Reply:
You're right. I'll update the type